internal/ansi/progress.go

```diff
@@ -0,0 +1,29 @@
+package ansi
+
+import (
+	"errors"
+	"github.com/schollz/progressbar/v3"
+)
+
+func ProgressBar[T comparable](desc string, items []T, fn func(int, T) error) error {
```

Comment:
```suggestion
// ProgressBar will display progress indication for the given items.
func ProgressBar[T comparable](desc string, items []T, fn func(int, T) error) error {
```

Let's make sure exported funcs are documented.